pkg/slack/slack.go

```diff
@@ -241,3 +242,20 @@ func (s *Notifier) composeTestMessage(execution *testkube.Execution, eventType t
 	}
 	return message.Bytes(), nil
 }
+
+// maskToken only shows the 4 characters of a slack token and masks the rest
+func maskToken(s string) string {
```

Comment:
It's quite generic function which should be in text package

Reply:
@exu I think `Obfuscate` is even better, makes for a much smaller change in this MR and you could recognise one slack token from another from the leading and trailing non-obfuscated characters imho.